web/src/views/SubmissionPortal/Components/IntroBlurb.vue

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+import {
+  defineComponent,
+} from '@vue/composition-api';
+
+export default defineComponent({
+  components: {},
+  setup() {
+    return {};
+  },
```

Comment:
remove

Reply:
Done in commit 794d5b95eef58a33da7c4cbe32cc04e40c6c2778.